sdk/nodejs/x/automation/localWorkspace.ts

```diff
@@ -384,10 +382,11 @@ export class LocalWorkspace implements Workspace {
      * @param config The `ConfigMap` to upsert against the existing config.
      */
     async setAllConfig(stackName: string, config: ConfigMap): Promise<void> {
-        // TODO: do this in parallel after this is fixed https://github.com/pulumi/pulumi/issues/3877
+        const configAdds: Promise<any>[] = [];
```

Comment:
Minor - but can this be?

```suggestion
        const configAdds: Promise<void>[] = [];
```